dsl/docker-multiarch/images/openjdk.groovy

```diff
@@ -0,0 +1,44 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+	meta = multiarch.meta(getClass(), arch)
+
+	freeStyleJob(meta.name) {
+		description(meta.description)
+		logRotator { daysToKeep(30) }
+		label(meta.label)
+		scm {
+			git {
+				remote { url('https://github.com/docker-library/openjdk.git') }
+				branches('*/master')
+				clean()
+			}
+		}
+		triggers {
+			upstream("docker-${arch}-debian, docker-${arch}-buildpack", 'UNSTABLE')
+			scm('H H/6 * * *')
+		}
+		wrappers { colorizeOutput() }
+		steps {
+			shell(multiarch.templateArgs(meta, ['dpkgArch']) + '''
+case "$dpkgArch" in
+	ppc64el)
+		sed -i 's!amd64!ppc64el!g' */Dockerfile
+		sed -i 's!unzip!unzip udev!' openjdk-9-jdk/Dockerfile
```

Comment:
I have removed these lines as I no longer see this error for openjdk9 for the subversion 9~b102. However now the latest subversion has been changed to 9~b107 in this commit:

https://github.com/docker-library/openjdk/commit/d7f632dcc518cf71752e95387d05cbb7f9156e1d

 but this version is not available yet for ppc64le so testing give package unavailable error for openjdk9

Not sure how to handle this but removing these lines for now as they are unnecessary anyways in this groovy script